test/gtest/uct/uct_test.cc

```diff
@@ -619,6 +632,11 @@ uct_test::entity* uct_test::create_entity(size_t rx_headroom,
                                      (ucs_empty_function_return_success) :
                                      rndv_cb;
     iface_params.rndv_arg          = rndv_arg;
+    iface_params.async_event_cb    = (async_event_cb == NULL) ?
+                                     reinterpret_cast<uct_async_event_cb_t>
+                                     (ucs_empty_function_return_success) :
```

Comment:
shouldn't it return void?
BTW maybe we can allow NULL on UCT API level, like we do for TAG callbacks for example?

Reply:
> shouldn't it return void?

no, it has `ucs_status_t` return value

yes, we allow. but just passing real CB to always test UD with this callback set.
see above, we passing for `rndv_cb` and `eager_cb` de3fault callbacks

Reply:
so why do we need return value? there isn't anything equivalent in event_fd right?
and if we allow, why not just pass async_event_cb  all the time (which may be NULL or real cb)?

Reply:
> so why do we need return value?

just for UCT user (e.g. UCP in the future) that will need to report an error thru the callback if something bad happened

> there isn't anything equivalent in event_fd right?

no, we don't have handler for event_fd. since it could get event thru poll/epoll_wait/select

> and if we allow, why not just pass async_event_cb all the time (which may be NULL or real cb)?

ok, let's pass `NULL` - done